cypress/integration/pages/index.js

```diff
@@ -25,20 +68,56 @@ const runCommonTests = ({ service, pageType }) => {
 
       if (pageType !== 'errorPage404') {
         it('should include the canonical URL', () => {
-          cy.checkCanonicalURL(
+          cy.get('head link[rel="canonical"]').should(
+            'have.attr',
+            'href',
             `https://www.bbc.com${config[service].pageTypes[pageType].path}`,
           );
         });
 
         it('should have a correct robot meta tag', () => {
-          cy.checkMetadataContent('head meta[name="robots"]', 'noodp,noydir');
+          cy.get('head meta[name="robots"]').should(
+            'have.attr',
+            'content',
+            'noodp,noydir',
+          );
         });
       }
+
+      it('should have lang and dir attributes', () => {
+        cy.request(`${config[service].pageTypes.articles.path}.json`).then(
```

Comment:
This has the articles path - it should be changed to vary on pageType
```suggestion
        cy.request(`${config[service].pageTypes[${pageType}].path}.json`).then(
```